xdsl/backend/mps/print_mps.py

```diff
@@ -0,0 +1,7 @@
+from typing import IO
+
+from xdsl.dialects.builtin import ModuleOp
+
+
+def print_to_mps(prog: ModuleOp, output: IO[str]):
+    raise NotImplementedError("MPS backend not yet implemented")
```

Comment:
Can you please add a short doc string for this function and module? Just a short sentence in each would be great.